apps/react-18-tests-v8/webpack.config.js

```diff
@@ -12,22 +12,54 @@ module.exports = () => {
     },
     resolve: {
       extensions: ['.tsx', '.ts', '.js'],
-      plugins: [
-        new TsconfigPathsPlugin({
-          configFile: path.resolve(__dirname, '../../tsconfig.base.json'),
-        }),
-      ],
       alias: {
         react: path.resolve(__dirname, './node_modules/react'),
         'react-dom': path.resolve(__dirname, './node_modules/react-dom'),
+        ...getResolveAlias(),
       },
     },
     module: {
       rules: [
         {
           test: /\.tsx?$/,
           exclude: /node_modules/,
-          use: 'swc-loader',
+          use: {
+            // NOTE: swc-loader throws errors on v8 codebase
```

Comment:
ups, turns out v8 code contains some invalid compiler constructs ( bug in rust parsing ? )

Reply:
https://github.com/microsoft/fluentui/issues/25767